.gitignore

```diff
@@ -1,3 +1,3 @@
 .DS_Store
 .DS_store
-.history
+.history
```

Comment:
Please add a newline to the end of this file.

Reply:
I was torn with that and wasn't sure so I created two. Your logic is good -- I can see someone doing that. I've merged the two into a single mod.

Thank you